backend/src/v2/compiler/argocompiler/container.go

```diff
@@ -527,38 +536,61 @@ func (c *workflowCompiler) getTaskRetryPolicySpec(name string) *pipelinespec.Pip
 	return taskSpec.RetryPolicy
 }
 
-func (c *workflowCompiler) getTaskRetryStrategy(name string) *wfapi.RetryStrategy {
+func (c *workflowCompiler) getTaskRetryParameters(name string) *[]wfapi.Parameter {
```

Comment:
We don't need to use a pointer here on the returned slice.
```suggestion
func (c *workflowCompiler) getTaskRetryParameters(name string) []wfapi.Parameter {
```

Reply:
Updated